migrations/versions/6e27124fd78c_.py

```diff
@@ -0,0 +1,30 @@
+"""Associate a user with a role.
+
+Revision ID: 6e27124fd78c
+Revises: 4938ca7b0713
+Create Date: 2020-09-29 17:41:41.027274
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '6e27124fd78c'
+down_revision = '4938ca7b0713'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
So, how do you want to handle assigning the `brighthive_admin` user a Role? 

I think it makes sense to do that with a script or manually....however, I still want to check: could this be done in a migration? Especially, since the `brighthive_admin` effectively gets "seeded" by an earlier migration, but the `brighthive_admin` would not be wholly useful without a Role. (Then again...our concept of Roles will change, so maybe we should not memorialize it in a migration.)

Reply:
Since a role is an optional thing, I can do with not assigning a role to that user for the moment. The idea is that you would log in as brighthive_admin and almost immediately disable or delete that account.

Reply:
Right, but you will not be able to log into Facet if the `brighthive_admin` does not have a Role. Sooo....you'd have to manually patch the `brighthive_admin`, after you spin up Authserver + Facet. Is that okay by you?